requirements-dev-frozen.txt

```diff
@@ -182,12 +182,13 @@ s3transfer==0.10.0
 selenium==4.16.0
 sentry-arroyo==2.16.5
 sentry-cli==2.16.0
+sentry-covdefaults-disable-branch-coverage==1.0.2
```

Comment:
Why did these packages change?

Reply:
it's an artifact of an out-of-date branch and GHA running on the merge base -- a rebase / merge will eliminate these "diffs"